orderly_install

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+if [[ $# -eq 0 ]]; then
+    echo "Usage: ./orderly_install instance_name 'command'
+    instance_name - fertility or naomi
+    command - The remotes::install_github or install_packages command, remember the quotes"
+    exit 0
+fi
+
+CONTAINERS=`docker ps --filter name=$1_orderly* --filter status=running -aq`
+CONTAINER_NAMES=`docker ps --filter name=$1_orderly* --filter status=running --format '{{.Names}}' -a | paste -sd '~ ' | sed 's/~/, /'`
+echo "Running Rscript in containers ${CONTAINER_NAMES}"
+
+for CONTAINER in $CONTAINERS
+do
+    docker exec $CONTAINER Rscript -e $2
```

Comment:
If you do `shift` after getting `$1` then you can pass everything though here as `$*` or `$@`